src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSubscriptionEditModel.ts

```diff
@@ -52,7 +59,35 @@ class ongoingTaskSubscriptionEditModel extends ongoingTaskEditModel {
         this.startingPointLatestDocument = ko.pureComputed(() => {
             return this.startingPointType() === "Latest Document";
         });
-    }   
+
+       this.changeVectorForNextBatchStartingPointFormatted = ko.pureComputed(() => {          
```

Comment:
can we introduce method which returns pure computed, as we repeat code here?

Reply:
done